### PR TITLE
test: cover stylometric signals and guard fixture coverage

### PR DESCRIPTION
**Pull request by conorbronsdon**
## Why
Four detector types had no direct fixtures, so a removed or over-broad signal could pass the suite. Aggregate score checks can conceal that failure when unrelated patterns still fire.

## Changes
- Add seven tests covering sentence and paragraph uniformity, bold overuse, confidence stacking before deduplication, low function-word trigram entropy, and its single-trigram edge case.
- Pair must-fire cases with ordinary-prose or adjacent-threshold must-not-fire controls; assert the specific type and emitted finding.
- Add a category-contract guard for named fixtures. Nine existing phrase-level gaps are explicit, fixed exceptions; a new missing type or removal of current coverage fails. An exception must be removed when its fixtures land.
- Document that this name-presence check is not runtime coverage and does not replace meaningful assertions.

Related to #242. This is its stylometric coverage and incremental guard slice. Complete coverage still depends on #213 and the outstanding phrase-level false-positive work; those newcomer tasks remain available. No detector behavior or generated skill source changes.

## Verification
- `npm test`: 20 suites passed, 0 failed.
- `npm run self-scan:check`: all document budgets pass.
- `bash scripts/check-pattern-count.sh` and `git diff --check`: pass.
- Six isolated mutations disabled each detector branch in turn; each caused its intended new fixture to fail.
- Two isolated contract mutations removed a fixture type name or introduced a new type; both failed the coverage guard.

Prepared by ChatGPT/Codex. Independent-family review through Claude/OpenCode is pending: those runtimes are unavailable in this session. The PR is prepared for review, not represented as independently approved. Tests-only maintenance needs no changelog entry under CONTRIBUTING.md.

### CODE REVIEW

**Comment by conorbronsdon** [comment]
Supplemental GPT-5.6 Sol review completed at exact head `88b4f2f57892818d0e20dd71c4762ff5e80a4817`. The reviewer matched all three changed-file blobs to GitHub, reran `npm test` (20 suites passed), and reported no actionable findings. Eight independent branch/threshold mutations were detected, including moving confidence gating after deduplication and changing the entropy sample boundary from 150 to 151 words.

This is supplemental same-family review. It does not replace the pending independent-family local review.

File: detector/categories.test.js [hunk 1]
```diff
@@ -66,6 +66,32 @@ test('every type referenced in the tables is a real detector type', () => {
...
+];
+
+test('every detector type has a named fixture or an explicit legacy gap', () => {
+  const fixtures = fs.readFileSync(path.join(__dirname, 'patterns.test.js'), 'utf8');
+  const missing = typeKeys.filter(type =>
+    !fixtures.includes(`'${type}'`) && !fixtures.includes(`"${type}"`)
+  ).sort();
+  assert.deepEqual(missing, LEGACY_UNCOVERED_TYPES,
+    'Fixture coverage changed: add a fixture for a new/missing type, or remove ' +
+    'a legacy exception whose fixture has landed. Do not expand the exception list.');
```
[the remark this answers is not in the document]
Comment (conorbronsdon) [reply detector/categories.test.js:92]: Verified, but keeping the ratchet as designed. The nine allowlisted types are the phrase-level fixtures that #213 reserves for a first-time contributor, plus the ones owned by the open false-positive issues, and CONTRIBUTING asks maintainers to leave those for newcomers. Filling them here would take that work away and widen a tests-only slice. The guard still does what #242's acceptance criteria ask: a new `TYPE_LABELS` key with no fixture fails, and removing any current fixture fails. The allowlist can only shrink, since `deepEqual` rejects both a tenth entry and a stale one.

Follow-up on branch `claude/pr-326-review-fixes-tow5np`: the same review found that the sentence-uniformity fixture used identical sentences (CV 0), so the 0.25 threshold was not pinned. That branch adds a bracketing pair (CV 0.243 fires, 0.286 stays clean) and replaces the hardcoded "Nine" in CONTRIBUTING with a reference to `LEGACY_UNCOVERED_TYPES` so the count cannot drift.

---
_Generated by [Claude Code](https://claude.ai/code)_
